django/http/response.py

```diff
@@ -408,6 +409,16 @@ def content(self, value):
             content = self.make_bytes(value)
         # Create a list of properly encoded bytestrings to support write().
         self._container = [content]
+        self._clear_cached_text()
+
+    @cached_property
+    def text(self):
+        encoding = self.charset or "utf-8"
+        return self.content.decode(encoding)
+
+    def _clear_cached_text(self):
+        if "text" in self.__dict__:
+            del self.__dict__["text"]
```

Comment:
```suggestion
        if (text := "text") in self.__dict__:
            del self.__dict__[text]
```
Another personal suggestion: in this case it's less error prone to use one variable than writing the same string twice.

Reply:
I also agree with this part as well.